pkgs/os-specific/linux/kernel/linux-rpi.nix

```diff
@@ -22,10 +23,21 @@ lib.overrideDerivation (buildLinux (args // {
     "4" = "bcm2711_defconfig";
   }.${toString rpiVersion};
 
+  extraConfig = ''
+    # ../drivers/pci/controller/pcie-altera.c:679:8: error: too few arguments to function 'devm_of_pci_get_host_bridge_resources'
+    PCIE_ALTERA n
+  '';
+
   features = {
     efiBootStub = false;
   } // (args.features or {});
 
+  extraConfig = ''
+    # From #97883. Testing to see if this fixes compile issue
```

Comment:
Is this comment still accurate? We should have tested for this to build, and if it does, remove it.

Reply:
Yeah, this is defined twice in this expression, which doesn't work.